AutomatedTesting/Gem/PythonTests/assetpipeline/fbx_tests/fbx_tests.py

```diff
@@ -758,7 +816,81 @@ def compare_scene_debug_file(asset_processor, expected_file_path, actual_file_pa
+        # are showing differences in hashes.
+
+        # If both scene graphs are identical, then bail, everything is working as expected.
+        if len(diff_actual) == 0 and len(diff_expected) == 0:
+            return None, None
+
+        # If one scene has more lines than the other, it's always an error
+        assert len(diff_actual) == len(diff_expected), "Scene mismatch - different line counts"
+
+        # If this is the XML debug file, then it will be difficult to verify if a line is a hash line or another integer.
+        # However, because XML files are always compared after standard dbgsg files, the hashes from that initial comparison can be used here to check.
+        is_xml_dbgsg = os.path.splitext(expected_file_path)[-1].lower() == ".xml"
+
+        if is_xml_dbgsg:
+            # If the difference count doesn't match the non-xml file, then it's not just a hash mis-match, fail.
+            assert len(expected_hashes_to_skip) == len(diff_expected), "Scene mismatch"
+            assert len(actual_hashes_to_skip) == len(diff_actual), "Scene mismatch"                
+
+            # This test did a simple line by line comparison, and didn't actually load the XML data into a graph to compare.
+            # Which means that the relevant info for this field to make it clear that it is a hash and not another number is not immediately available.
+            # So instead, extract the number and compare it to the known list of hashes.
+            # If this regex fails or the number isn't in the hash list, then it means this is a non-hash difference and should cause a test failure.
+            # Otherwise, if it's just a hash difference, it can be a warning for now, while the information being hashed is not stable across platforms.
+            xml_number_regex = re.compile('.*<Class name="AZ::u64" field="m_data" value="([0-9]*)" type="{D6597933-47CD-4FC8-B911-63F3E2B0993A}"\\/>')
+
+            for list_entry in diff_actual:
+                match_result = xml_number_regex.match(list_entry)
+                assert match_result, "Scene mismatch"
+                data_value = match_result.group(1)
+                # This value doesn't match the list of known hash differences, so mark this test as failed.
+                assert (data_value in actual_hashes_to_skip), "Scene mismatch"
+                
+            for list_entry in diff_expected:
+                match_result = xml_number_regex.match(list_entry)
+                assert match_result, "Scene mismatch"
+                data_value = match_result.group(1)
+                # This value doesn't match the list of known hash differences, so mark this test as failed.
+                assert (data_value in expected_hashes_to_skip), "Scene mismatch"
+
+            return expected_hashes_to_skip, actual_hashes_to_skip
+        else:
+            # The last check is to see if the difference is just hash mismatch.
```

Comment:
Do we want to tie this error downgrade to a GHI so its clear why?

Reply:
I was going to do that later, I'm not sure how I want to form those tickets yet. I just closed out a bunch of very old "Investigate X before it causes problems later" tickets, so I don't want to create another one that we'll just close in a few years.